apps/condo/domains/document/schema/Document.js

```diff
@@ -53,6 +53,12 @@ const Document = new GQLListSchema('Document', {
             isRequired: true,
             access: createAndReadOnlyFieldAccess,
         },
+        isPublic: {
```

Comment:
1) We have `canReadByResident` field in `Ticket` with same logic. Maybe we should name this field the same?
2) `Document` read accesses need to be updated for residents